### PR TITLE
Create project structure:

### PR DESCRIPTION
**Pull request by Devour2005**
Create project structure: 
-add the entities;  
-add the layers;
-add dependencies;

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile group: 'org.springframework', name: 'spring-jdbc', version: '4.3.12.RELEASE'
```
Comment (Andriireva) [inline build.gradle]: Use the same structure. Like in the implementation

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter'
+    compileOnly 'org.projectlombok:lombok'
```
Comment (Andriireva) [inline build.gradle]: to make it more readable lets separate via empty line  implementation , compileOnly , developmentOnly ....

File: src/main/java/com/employeeregistry/task/domain/Employee.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Employee implements Serializable {
+
+  private static final long serialVersionUID = 1L;
+
+  private Long employeeId;
```
Comment (Andriireva) [inline src/main/java/com/employeeregistry/task/domain/Employee.java]: usually it does not make sense to add name of the class to the field name. Lets simplify it to "id"

File: src/main/java/com/employeeregistry/task/domain/Organization.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Organization implements Serializable {
+
+  private static final long serialVersionUID = 1L;
+
+  private Long orgId;
```
Comment (Andriireva) [inline src/main/java/com/employeeregistry/task/domain/Organization.java]: lets simplify it to id

File: src/main/java/com/employeeregistry/task/repositiry/EmployeeRepository.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+@Repository
+public class EmployeeRepository implements RegistryRepository<Employee> {
+
+  private JdbcTemplate jdbcTemplate;
+
+  public void setJdbcTemplate(JdbcTemplate jdbcTemplate) {
+    this.jdbcTemplate = jdbcTemplate;
+  }
+
```
Comment (Andriireva) [inline src/main/java/com/employeeregistry/task/repositiry/EmployeeRepository.java]: remove empty lines

File: src/main/java/com/employeeregistry/task/service/RegistryService.java [hunk 6]
```diff
@@ -0,0 +1,5 @@
+package com.employeeregistry.task.service;
+
+public interface RegistryService {
```
Comment (Andriireva) [inline src/main/java/com/employeeregistry/task/service/RegistryService.java]: From my point of view this interface is useless. But this change is optional.

File: build.gradle (hunk 1 above)
Comment (Devour2005) [reply build.gradle]: Done

File: build.gradle (hunk 2 above)
Comment (Devour2005) [reply build.gradle]: Done

File: src/main/java/com/employeeregistry/task/domain/Employee.java (hunk 3 above)
Comment (Devour2005) [reply src/main/java/com/employeeregistry/task/domain/Employee.java]: Changed.

File: src/main/java/com/employeeregistry/task/domain/Organization.java (hunk 4 above)
Comment (Devour2005) [reply src/main/java/com/employeeregistry/task/domain/Organization.java]: Changed.

**Pull request merged**